modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,603 @@
+=== {tekton-chains}
+
+* With this update, when using `docdb` storage in {tekton-chains}, you can configure the `MONGO_SERVER_URL` value directly in the `TektonConfig` CR as the `storage.docdb.mongo-server-url` setting. Alternatively, you can provide this value using a secret and set the `storage.docdb.mongo-server-url-dir` setting to the directory where the `MONGO_SERVER_URL` file is located.
++
+.Example of creating a secret with the `MONGO_SERVER_URL` value
+[source,terminal]
+----
+$ oc create secret generic mongo-url -n tekton-chains \ #
+  --from-file=MONGO_SERVER_URL=/home/user/MONGO_SERVER_URL 
+----
++
+.Example of configuring the `MONGO_SERVER_URL` value using a secret
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+# ...
+  chain:
+    disabled: false
+    storage.docdb.mongo-server-url-dir: /tmp/mongo-url
+    options:
+      deployments:
+        tekton-chains-controller:
+          spec:
+            template:
+              spec:
+                containers:
+                - name: tekton-chains-controller
+                  volumeMounts:
+                  - mountPath: /tmp/mongo-url
+                    name: mongo-url
+                volumes:
+                -  name: mongo-url
+                   secret:
+                    secretName: mongo-url
+# ...
+----
+
+* With this update, when using KMS signing in {tekton-chains}, instead of providing the KMS authentication token value directly in the configuration, you can provide the token value as a secret by using the `signers.kms.auth.token-path` setting.  
++
+[source,terminal]
+----
+$ oc create secret generic <secret_name> -n tekton-chains \
+  --from-file=KMS_AUTH_TOKEN=/home/user/KMS_AUTH_TOKEN <1>
+----
+<1> Replace `<secret_name>` with any name. The following example uses KMS secret called `kms-secrets`.
++
+.Example configuration of the KMS token value using a secret called `kms-secrets`
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+# ...
+  chain:
+    disabled: false
+    signers.kms.auth.token-path: /etc/kms-secrets/KMS_AUTH_TOKEN
+    options:
+      deployments:
+        tekton-chains-controller:
+          spec:
+            template:
+              spec:
+                containers:
+                - name: tekton-chains-controller
+                  volumeMounts:
+                  - mountPath: /etc/kms-secrets
+                    name: kms-secrets
+                volumes:
+                -  name: kms-secrets
+                   secret:
+                    secretName: kms-secrets
+# ...
+----
+
+* With this update, you can configure a list of namespaces as an argument to the {tekton-chains} controller. If you provide this list, {tekton-chains} watches pipeline runs and task runs only in the specified namespaces. If you do not provide this list, {tekton-chains} watches pipeline runs and task runs in all namespaces.
```

Comment:
```suggestion
* With this update, you can configure a list of namespaces as an argument to the {tekton-chains} controller. If you provide this list, {tekton-chains} watches pipeline runs and task runs only in the specified namespaces. If you do not provide this list, {tekton-chains} watches pipeline runs and task runs in all of the namespaces.
```

Reply:
Based on minimalism, I think all namespaces is a better option and also sounds more formal to me. So I will it as it is.